libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/guidance/junction/view/MapboxJunctionView.kt

```diff
@@ -43,7 +44,26 @@ class MapboxJunctionView : AppCompatImageView {
     ) : super(context, attrs, defStyleAttr)
 
     /**
-     * Invoke to render the signboard based on data or error conditions.
+     * Invoke to render the junction based on data or error conditions.
+     * A [callback] is triggered holding the junction in the form of [Bitmap]
+     */
+    fun render(
+        result: Expected<JunctionValue, JunctionError>,
+        callback: MapboxNavigationConsumer<Bitmap?>
```

Comment:
Wondering if we should return an `Expected` here instead of a nullable `Bitmap`? 🤔

Reply:
I don't see the need as it's a simple primitive type data structure. To wrap it inside `Expected`, it would need to introduce `Value` and `Error` model classes that would contain the Bitmap thereby adding an unnecessary extra step.